### ISSUE TITLE
tests fail with non-UTF8 systems

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/epitrix#25]
From the PR:
https://github.com/reconhub/epitrix/pull/24

We see issues with non-UTF8 systems causing CI services to fail; from `devtools::check()`:

```r
 checking examples ... ERROR
  Running examples in 'epitrix-Ex.R' failed
  The error most likely occurred in:
  
  > base::assign(".ptime", proc.time(), pos = "CheckExEnv")
  > ### Name: clean_labels
  > ### Title: Standardise labels
  > ### Aliases: clean_labels
  > 
  > ### ** Examples
  > 
  > 
  > clean_labels("-_-This is; A    WeÏrD**./sêntënce...")
  [1] "this_is_a_weird_sentence"
  > clean_labels("-_-This is; A    WeÏrD**./sêntënce...", sep = ".")
  [1] "this.is.a.weird.sentence"
  > <ERROR: re-encoding failure from encoding 'UTF-8'>
  input <- c("
  + input
  + clean_labels(input)
  + 
  + # Don't transliterate non-latin words
  + clean_labels(input, transformation = "Latin-ASCII")
  Error: unexpected symbol in:
  "# Don't transliterate non-latin words
  clean_labels(input, transformation = "Latin"
  Execution halted
```

**Comment by zkamvar** [comment]
I'll see what I can do. Character encoding is a PITA everywhere. 

FWIW, the CI in this instance that is failing is appveyor (surprise, surprise): https://ci.appveyor.com/project/thibautjombart/epitrix/builds/27590600/job/jbi2e40em4l2cj43#L1068

**Comment by thibautjombart** [comment]
@stephlocke it would be great if Lockdata could look into this issue

**Comment by stephlocke** [comment]
We can add this to the TODO!  @zkamvar I can't do any assigning here - could you see if Team LD can be elevated access-wise to support issue management

**Comment by thibautjombart** [comment]
This issue was currently occuring during the LaTeX conversion of the manual:
```
* checking PDF version of manual ... WARNING
LaTeX errors when creating PDF version.
This typically indicates Rd problems.
LaTeX errors found:
! LaTeX Error: Unicode character ま (U+307E)
               not set up for use with LaTeX.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
! LaTeX Error: Unicode character す (U+3059)
               not set up for use with LaTeX.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
! LaTeX Error: Unicode character だ (U+3060)
               not set up for use with LaTeX.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
! LaTeX Error: Unicode character よ (U+3088)
               not set up for use with LaTeX.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
! LaTeX Error: Unicode character し (U+3057)
               not set up for use with LaTeX.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
! LaTeX Error: Unicode character ひ (U+3072)
               not set up for use with LaTeX.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
! LaTeX Error: Unicode character こ (U+3053)
               not set up for use with LaTeX.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
* checking PDF version of manual without index ... ERROR
* skipping checking HTML version of manual: no command ‘tidy’ found
* checking for non-standard things in the check directory ... NOTE
Found the following files/directories:
  ‘epitrix-manual.tex’
* checking for detritus in the temp directory ... OK
* DONE

Status: 1 ERROR, 1 WARNING, 2 NOTEs
```
Removing kanjis (pardon my ignorance if they were not) in the example seems to fix the issue. After trying to only run this example on some platforms (and failing to fix), I am enclined to remove this line of the example.

**Comment by thibautjombart** [comment]
Closed via 2b338292b877f8b9a7076e50d08a3c69091344bf
